core/src/components/checkbox/test/basic/checkbox.e2e.ts

```diff
@@ -5,57 +5,58 @@ test.describe('checkbox: basic', () => {
   test('should not have visual regressions', async ({ page }) => {
     await page.goto(`/src/components/checkbox/test/basic`);
 
-    await page.setIonViewport();
-
-    expect(await page.screenshot()).toMatchSnapshot(`checkbox-basic-${page.getSnapshotSettings()}.png`);
+    const content = page.locator('#checkboxes');
```

Comment:
Can we also have a `color` test similar to https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/toggle/test/color/toggle.e2e.ts?